main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"gopkg.in/go-playground/webhooks.v5/github"
+	"net/http"
+)
+
+const (
+	path = "/payload"
+)
+
+func main() {
+	hook, _ := github.New(github.Options.Secret("<GITHUBSECRET>"))
```

Comment:
What return value are you ignoring here?

Also, what’s your plan for loading the secret?

Reply:
Is “hook” thread safe?

Reply:
The error being ignored is if `New` fails to allocated the memory or initialize the webhook instance. What do you mean by "thread safe?"